app/javascript/modules/listingDetails/ListingDetailsFeatures.tsx

```diff
@@ -38,6 +46,42 @@ const FeatureItem = ({ content, title, toTranslate }: FeatureItemProps) => {
 }
 
 export const ListingDetailsFeatures = ({ listing, imageSrc }: ListingDetailsFeaturesProps) => {
+  const [units, setUnits] = useState({})
+
+  useEffect(() => {
+    void getUnits(listing.Id).then((units) => {
+      // eslint-disable-next-line unicorn/no-array-reduce
```

Comment:
this no reduce rule seemed to come with an external linting ruleset. happy to try a different approach if folks find it difficult to read*

*imo I hate `.reduce`, but find that it's a symptom of data that hasn't been massaged properly for the front end

Reply:
ha I kind of like reduce. Even though I don't mind it, I feel similarly, if @christine-sfg isn't a fan I'd say find a different way!